src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerBulkRecord.java

```diff
@@ -15,81 +15,7 @@
  * 
  * This interface is implemented by {@link SQLServerBulkCommon} Class
  */
-public interface ISQLServerBulkRecord extends Serializable {
-
-    /**
-     * Returns the ordinals for each of the columns represented in this data record.
-     * 
-     * @return Set of ordinals for the columns.
-     */
-    public java.util.Set<Integer> getColumnOrdinals();
-
-    /**
-     * Returns the name of the given column.
-     * 
-     * @param column
-     *        Column ordinal
-     * @return Name of the column
-     */
-    public String getColumnName(int column);
-
-    /**
-     * Returns the JDBC data type of the given column.
-     * 
-     * @param column
-     *        Column ordinal
-     * @return JDBC data type of the column
-     */
-    public int getColumnType(int column);
-
-    /**
-     * Returns the precision for the given column.
-     * 
-     * @param column
-     *        Column ordinal
-     * @return Precision of the column
-     */
-    public int getPrecision(int column);
-
-    /**
-     * Returns the scale for the given column.
-     * 
-     * @param column
-     *        Column ordinal
-     * @return Scale of the column
-     */
-    public int getScale(int column);
-
-    /**
-     * Returns whether the column represents an identity column.
-     * 
-     * @param column
-     *        Column ordinal
-     * @return True if the column is an identity column; false otherwise.
-     */
-    public boolean isAutoIncrement(int column);
-
-    /**
-     * Returns the data for the current row as an array of Objects.
-     * 
-     * Each Object must match the Java language Type that is used to represent the indicated JDBC data type for the
-     * given column. For more information, see 'Understanding the JDBC Driver Data Types' for the appropriate mappings.
-     * 
-     * @return The data for the row.
-     * @throws SQLServerException
-     *         If there are any errors in obtaining the data.
-     */
-    public Object[] getRowData() throws SQLServerException;
-
-    /**
-     * Advances to the next data row.
-     * 
-     * @return True if rows are available; false if there are no more rows
-     * @throws SQLServerException
-     *         If there are any errors in advancing to the next row.
-     */
-    public boolean next() throws SQLServerException;
-
+public interface ISQLServerBulkRecord extends ISQLServerBulkData {

```

Comment:
We can remove `import java.io.Serializable;` on line 8 as we no longer need this.

Reply:
I'll remove